api-tests/apply_route_test.go

```diff
@@ -0,0 +1,216 @@
+package api_tests
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/Percona-Lab/percona-version-service/client"
+	"github.com/Percona-Lab/percona-version-service/client/models"
+	"github.com/Percona-Lab/percona-version-service/client/version_service"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_apply_should_return_just_one_version(t *testing.T) {
```

Comment:
Go doesn't use camel_case in function names

Reply:
https://github.com/Percona-Lab/percona-version-service/pull/12/commits/0c336ea232130725515cf4a0c0516190fbe5e510